packages/util/src/environment.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Node.js environment configurations.
+ */
+export enum Environment {
+  /**
+   * Production environment.
+   *
+   * Node.js assumes it's always running in a development environment. You can signal Node.js that you are running
+   * in production by setting the NODE_ENV=production environment variable.
+   */
+  Production = 'production',
+
+  /**
+   * Development environment.
+   *
+   * You can signal Node.js that you are running by setting the NODE_ENV=development environment variable.
+   */
+  Development = 'development',
+
+  /**
+   * Unknown environment.
+   */
+  Unknown = 'Unknown'
+}
+
+export const isProductionEnvironment = (): boolean => process.env.NODE_ENV === Environment.Production;
+export const isDevelopmentEnvironment = (): boolean => process.env.NODE_ENV === Environment.Production;
```

Comment:
No NODE_ENV is typically interpreted as development. I'd remove the Unknown variant too.
```suggestion
export const isDevelopmentEnvironment = (): boolean => process.env.NODE_ENV !== Environment.Production;
```